.github/workflows/docker.yml

```
@@ -5,6 +5,7 @@ on:
         branches:
             - unstable
             - stable
+            - deneb-free-blobs
```

```suggestion
            # TODO(sean): revert some/all changes in this file.
            - deneb-free-blobs
```

Not sure if this is helpful, but it might help when you go grepping for `TODO(sean)` before merging.